frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -516,13 +516,18 @@ class BitBox02 extends Component<Props, State> {
               )}
             </ViewContent>
             <ViewButtons>
-              <Button
-                primary
-                onClick={() => this.channelVerify(true)}
-                disabled={!deviceVerified}
-                hidden={status === 'pairingFailed'}>
-                {t('button.continue')}
-              </Button>
+              {deviceVerified ? (
+                <Button
+                  primary
+                  onClick={() => this.channelVerify(true)}
+                  hidden={status === 'pairingFailed'}>
+                  {t('button.continue')}
+                </Button>
+              ) : (
+                status !== 'pairingFailed' ? (
```

Comment:
could move this to the top and put everything in this condition (both the continue button and the PointToBitBox02)